packages/eslint-config-basic/index.js

```diff
@@ -4,133 +4,190 @@ module.exports = {
+            pathPattern: '^(?:dev|peer|optional|bundled)?[Dd]ependencies$',
+            order: { type: 'asc' },
+          },
+        ],
+      },
+    },
+  ],
   rules: {
     // import
-    "simple-import-sort/sort": "error",
-    "sort-imports": "off",
-    "import/first": "error",
-    "import/newline-after-import": "error",
-    "import/no-duplicates": "error",
-    "import/no-mutable-exports": "error",
-    "import/no-unresolved": "off",
-    "import/no-absolute-path": "off",
+    'simple-import-sort/sort': 'error',
+    'sort-imports': 'off',
+    'import/first': 'error',
+    'import/newline-after-import': 'error',
+    'import/no-duplicates': 'error',
+    'import/no-mutable-exports': 'error',
+    'import/no-unresolved': 'off',
+    'import/no-absolute-path': 'off',
 
     // Common
-    semi: [2, "never"],
-    curly: [2, "multi-or-nest", "consistent"],
-    quotes: ["error", "single"],
-    "no-unused-vars": "warn",
-    "no-param-reassign": "off",
-    "array-bracket-spacing": ["error", "never"],
-    "brace-style": ["error", "stroustrup", { allowSingleLine: true }],
-    "block-spacing": ["error", "always"],
-    camelcase: "off",
-    "comma-spacing": ["error", { before: false, after: true }],
-    "comma-style": ["error", "last"],
-    "comma-dangle": ["error", "always-multiline"],
-    "no-constant-condition": "warn",
-    "no-debugger": "warn",
-    "no-console": "warn",
-    "no-cond-assign": ["error", "always"],
-    "func-call-spacing": ["off", "never"],
-    "key-spacing": ["error", { beforeColon: false, afterColon: true }],
-    indent: [
-      "error",
-      2,
-      { SwitchCase: 1, VariableDeclarator: 1, outerIIFEBody: 1 },
-    ],
-    "no-restricted-syntax": [
-      "error",
-      "DebuggerStatement",
-      "ForInStatement",
-      "LabeledStatement",
-      "WithStatement",
+    'semi': ['error', 'never'],
+    'curly': ['error', 'multi-or-nest', 'consistent'],
+    'quotes': ['error', 'single'],
+    'quote-props': ['error', 'consistent-as-needed'],
+    'no-unused-vars': 'warn',
+    'no-param-reassign': 'off',
+    'array-bracket-spacing': ['error', 'never'],
+    'brace-style': ['error', 'stroustrup', { allowSingleLine: true }],
+    'block-spacing': ['error', 'always'],
+    camelcase: 'off',
+    'comma-spacing': ['error', { before: false, after: true }],
+    'comma-style': ['error', 'last'],
+    'comma-dangle': ['error', 'always-multiline'],
+    'no-constant-condition': 'warn',
+    'no-debugger': 'warn',
+    'no-console': ['error', { allow: ['warn', 'error'] }],
+    'no-cond-assign': ['error', 'always'],
+    'func-call-spacing': ['off', 'never'],
+    'key-spacing': ['error', { beforeColon: false, afterColon: true }],
+    'indent': ['error', 2, { SwitchCase: 1, VariableDeclarator: 1, outerIIFEBody: 1 }],
+    'no-restricted-syntax': [
+      'error',
+      'DebuggerStatement',
+      'ForInStatement',
+      'LabeledStatement',
+      'WithStatement',
     ],
-    "no-spaced-func": "error",
```

Comment:
why was this removed?

Reply:
> really hard to review as changing quotes and content at the same time..
> 
> in addition resoning for changing various values is missing... what are the changes doing and why?

well, you should review #3 first as written in the description of this PR.

Reply:
that's why https://github.com/mineiros-io/eslint-config/pull/3 is opened as a separate PR - it provides all the changes separately from this PR that only changes quotes